api/src/pcapi/core/history/api.py

```diff
@@ -41,21 +33,23 @@ def log_action(
     if not any((user, offerer, venue, finance_incident, bank_account, rule)) and (action_type not in legit_actions):
         raise ValueError("No resource (user, offerer, venue, finance incident, bank account, rule)")
 
-    if save:
-        if user is not None and user.id is None:
-            raise RuntimeError("Unsaved user would be saved with action: %s" % (user.email,))
+    # author/user/offerer/venue/bank_account object and its would may not be associated before flush() or commit()
```

Comment:
[détail] Ça ne date pas de ce commit, mais pendant qu'on y est : 

```suggestion
    # author/user/offerer/venue/bank_account object and its relations would not be associated before flush() or commit()
```